### PR TITLE
Fix county sharing and About page typo

### PR DESCRIPTION
**Pull request by schlosser**
I wasn't sure where to put the `County` type declaration.

### CODE REVIEW

File: src/components/LocationPage/ChartsHolder.tsx [hunk 1]
```diff
@@ -158,13 +172,13 @@ const ChartsHolder = (props: {
             </MainContentInner>
             <ClaimStateBlock
               stateId={props.stateId}
-              countyId={props.countyId}
+              countyId={props.county.county_url_name}
```
Comment (mikelehen) [inline src/components/LocationPage/ChartsHolder.tsx:175]: Looks like the state pages are broken. 😬  https://covid-projections-git-county-share.covidactnow.now.sh/us/wa

`props.county?.county_url_name` might fix it?

File: src/components/LocationPage/ChartsHolder.tsx [hunk 2]
```diff
@@ -69,7 +80,10 @@ const ChartsHolder = (props: {
   return (
     <>
       {!projection ? (
-        <NoCountyDetail countyId={props.countyId} stateId={props.stateId} />
+        <NoCountyDetail
+          countyId={props.county.county_url_name}
```
Comment (mikelehen) [inline src/components/LocationPage/ChartsHolder.tsx:84]: This code probably only gets hit for counties, but to be safe, `props.county?.county_url_name` here too.

File: src/components/LocationPage/ChartsHolder.tsx [hunk 3]
```diff
@@ -43,10 +43,21 @@ import {
...
+type County = {
+  county: string;
+  county_url_name: string;
+  county_fips_code: string;
+  state_fips_code: string;
+  state_code: string;
+  full_fips_code: string;
+  cities: string[];
+  population: string;
+};
```
Comment (mikelehen) [inline src/components/LocationPage/ChartsHolder.tsx:55]: Thank you for this! 😁  I'm not sure where it should go either.  I feel like we probably need a util/counties.ts file with this type and some helpers for dealing with the county dataset.  Maybe add a TODO, and I might try to grab it as tech debt later this week.

File: src/components/LocationPage/ChartsHolder.tsx (hunk 1 above)
Comment (schlosser) [reply src/components/LocationPage/ChartsHolder.tsx]: Updated!

File: src/components/LocationPage/ChartsHolder.tsx (hunk 3 above)
Comment (schlosser) [reply src/components/LocationPage/ChartsHolder.tsx:55]: Added a TODO!

File: src/components/LocationPage/ChartsHolder.tsx (hunk 2 above)
Comment (schlosser) [reply src/components/LocationPage/ChartsHolder.tsx]: Done!

**Comment by schlosser** [comment]
Added code to put the right name: `countyName, countyCode`

**Pull request merged**